### PR TITLE
Alpine build failure

### PR DESCRIPTION
**Pull request by elifaslan1**
### Description
Addresing Alpine build failure.
Whole patch is too large to backport so I am only fixing this line to pass build issue

### Related issues
https://bugs.openjdk.org/browse/JDK-8301493

### How has this been tested?

Tested on mac , generic linux and alpine aarch 64

### CODE REVIEW

**Comment by benty-amzn** [comment]
This looks like a partial backport of https://github.com/openjdk/jdk/commit/948f3b3c24709eca3aa6c3f0db6adb9226d6f9ac#diff-1ff58ce70aeea7e9842d34e8d8fd9c94dd91182999d455618b2a171efd8f742c, is there a reason we're doing this instead of taking the upstream patch?

**Comment by elifaslan1** [comment]
I asked maintainer what they can do here:https://bugs.openjdk.org/browse/JDK-8301493

**Review by mrserb — APPROVED** [review]

**Comment by mrserb** [comment]
looks fine as a temporary solution to unblock the build issue.

**Pull request merged**
